composition-js/src/merging/merge.ts

```diff
@@ -295,15 +295,15 @@ class Merger {
 
     for (const type of this.merged.types()) {
       // We've already merged unions above
-      if (type.kind === 'UnionType' || joinSpec.isSpecType(type)) {
+      if (type.kind === 'UnionType' || [linkSpec, joinSpec].some((s) => s.isSpecType(type))) {
```

Comment:
**style preference** To mine eyes, this is still a little easier to understand in a hurry

```suggestion
      if (type.kind === 'UnionType' || linkSpec.isSpecType(type) || joinSpec.isSpecType(type) {
```

Reply:
Agreed, it reads a tad better. Thanks.

Reply:
omg except I'm the worst and forgot the trailing `)`

😞

Reply:
Hahah. That's ok, I'll fix.

Reply:
You didn't _have_ to, you just _like_ to 😸